packages/core/integration-tests/test/parcel-register.js

```diff
@@ -0,0 +1,109 @@
+// @flow strict-local
+
+import {execSync} from 'child_process';
+import assert from 'assert';
+import path from 'path';
+
+describe.skip('@parcel/register', () => {
```

Comment:
why are they skipped?

Reply:
From the test plan:

> Since @parcel/require uses parcel to transform and resolve, it requires a pre-built version of Parcel and cannot be tested without this (I developed one to test this locally).

Parcel register requires parcel, and I wasn't daring enough to try to combine `@babel/register` and `@parcel/register`.